src/support/components/PayGSupportOverlay.tsx

```diff
@@ -96,20 +100,34 @@ const PayGSupportOverlay: FC<OwnProps> = () => {
   }
 
   const handleSubmit = async () => {
-    const email = quartzMe?.email
-    const descriptionWithOrgId = `${description}  [Org Id: ${orgID}]`
+    let userID: string, userEmail: string, orgName: string, orgID: string
+
+    if (shouldUseQuartzIdentity()) {
+      userID = identityUser.id
+      userEmail = identityUser.email
+      orgName = identityOrg.name
+      orgID = identityOrg.id
+    } else {
+      // Optional chaining operator needed because quartzMe's initial reducer state is null.
+      userID = quartzMe?.id
+      userEmail = quartzMe?.email
+      orgName = quartzOrg?.name
+      orgID = quartzOrg?.id
+    }
+
+    const descriptionWithOrgId = `[Org Name: ${orgName}] [Org Id: ${orgID}] \n\n ${description}`
```

Comment:
Added two newlines between the org name/id and description to make the ticket description read more cleanly.

Reply:
Vereesh asked me to put the org id at the end of the description. Has that changed?

Reply:
As discussed - I've reverted this to the existing location in the description until told otherwise.